http/oidc/src/main/java/org/wildfly/security/http/oidc/OidcJsonConfiguration.java

```diff
@@ -152,13 +157,21 @@ public class OidcJsonConfiguration {
     protected int tokenMinimumTimeToLive = 0;
     @JsonProperty(MIN_TIME_BETWEEN_JWKS_REQUESTS)
     protected int minTimeBetweenJwksRequests = 10;
+    @JsonProperty(POST_LOGOUT_URI)
+    protected String postLogoutUri;
     @JsonProperty(PUBLIC_KEY_CACHE_TTL)
     protected int publicKeyCacheTtl = 86400; // 1 day
     // https://tools.ietf.org/html/rfc7636
     @JsonProperty(ENABLE_PKCE)
     protected boolean pkce = false;
     @JsonProperty(IGNORE_OAUTH_QUERY_PARAMETER)
     protected boolean ignoreOAuthQueryParameter = false;
+    @JsonProperty(LOGOUT_PATH)
+    protected String logoutPath;
```

Comment:
Could set the default value here and below.

Reply:
Done